components/persistor/lib/knex/db.ts

```diff
@@ -805,41 +820,52 @@ module.exports = function (PersistObjectTemplate) {
 
                     name = name.toLowerCase();
                     if (operation === 'add') {
-                        table[type](columns, name);
+                        await knex.schema.table(tableName, function (table) {
+                            table[type](columns, name);
+                        }).catch((error) => {
```

Comment:
await should be wrapped in a try catch, await X.catch is mixing 2 paradigms.

Reply:
changed